implement-directus-auth-in-nextjs-14/index.md

```diff
@@ -0,0 +1,995 @@
+---
+title: 'Implement Directus Auth in Next.js 14'
+description: 'A tutorial on implementing Authentication on a Next.js blog website using Directus as a Headless CMS.'
+author:
+  name: 'Emmanuel John'
+  avatar_file_name: 'profile-photo.jpeg'
+---
+
+## Introduction
+In this tutorial, you will learn how to implement Authentication on a Next.js blog website using Directus as a Headless CMS. You’ll register new users, login, handle refresh tokens, create posts, list all posts, and edit/delete posts where the auth user is the post creator.
+
+## Before You Start
+To follow along with this tutorial, you will need the following knowledge and tools:
+
+- [Node.js](https://nodejs.org/en/download/) v18 and above installed on your computer.
+- A Directus project - follow our [quickstart guide](https://docs.directus.io/blog/docs.directus.io/getting-started/quickstart.html) if you don't already have one.
+- Prior experience with Next.js.
+
+<!-- ## Your Sections Here -->
+## Allowing Public Role To Create Users
+
+1. Navigate to `Settings`.
+2. Select `Access Control`.
+3. Click on the `Public Role`.
+4. Access the `System Collection`.
+5. Locate `directus_users`.
+6. Under the `Create` option, choose `Use Custom`.
+7. Enable field permissions for `first_name`, `last_name`, `role`, `Email` and `Password` options.
+![Directus public role field permission settings](public-role-field-permission-settings.png)
+8. Enable field validation where `role` equals the Author role `Primary Key`
+![Directus public role field validation settings](public-role-field-validation-settings.png)
+
+## Creating posts collection
+
+1. Navigate to `Settings`.
+2. Select `Data Model`.
+3. Create a new collection named `posts`.
+4. Check the fields that you want returned on the `posts` data
+![Directus posts optional fields settings](Directus-posts-optional-fields-settings.png)
+5. Create additional fields for the post:
+
+- title => Input field
+- content => WYSIWYG
+- image => Image
+The `title` and `content` fields should be required.
+![Directus post additional fields settings](Directus-post-additional-fields-settings-1.png)
```

Comment:
```suggestion
## Creating a Posts Collection

Create a new `posts` collection. Ensure `user_created` is created as an optional field as it will be used to lock down permissions so a user can only edit or delete their own posts. Create the following additional fields:

- `title`: a text input field
- `content`: a WYSIWYG field
- `image`: an image field

- title => Input field
- content => WYSIWYG
- image => Image
```

Required fields, I think, are not strictly required by this project so lets drop the instruction.